File: 35/Pallamano.py
from Squadra import Squadra
class Pallamano(Squadra):
    def __init__(self, cod, tipo, nomeSquadra, numPartiteVinte, numPartitePerse,nMedioDiReti) -> None:
        super().__init__(cod, tipo, nomeSquadra, numPartiteVinte, numPartitePerse)
        self.nMedioDiReti = nMedioDiReti
    
        #nome della squadra, codice, n. partite vinte, n. partite perse,n. medio di gol, n. medio di falli, n. medio di reti, sport
    
    def __str__(self) -> str:
        return self.nomeSquadra+"\t"+str(self.cod)+"\t"+str(self.numPartiteVinte)+"\t"+str(self.numPartitePerse)+"\t"+"-"+"\t"+"-"+"\t"+str(self.nMedioDiReti)+"\t"+self.tipo

File: simu16/src/Pallamano.java
public class Pallamano extends Squadra {
    double n_medio_di_reti;

    public Pallamano(int codice, String sport, String nome_della_squadra, int num_partite_vinte, int num_partite_perse,
            double n_medio_di_reti) {
        super(codice, sport, nome_della_squadra, num_partite_vinte, num_partite_perse);
        this.n_medio_di_reti = n_medio_di_reti;
    }

    @Override
    public String toString() {
        return nome_della_squadra + "\t" + codice + "\t" + num_partite_vinte + "\t" + num_partite_perse + "\t"+ "-" + "\t" + "-" + "\t" + n_medio_di_reti+"\t" + sport;
    }

}
